rats-tls/samples/enclave-tls-client/client.c

```diff
@@ -5,19 +5,29 @@
  */
 
 #include <stdlib.h>
+#include <ctype.h>
 #include <stdio.h>
 #include <string.h>
 #include <strings.h>
 #include <getopt.h>
+#include <stdbool.h>
 #include <sys/socket.h>
 #include <arpa/inet.h>
 #include <netinet/in.h>
 #include <unistd.h>
 #include <enclave-tls/api.h>
 #include <enclave-tls/log.h>
+#include <enclave-tls/utils.h>
 
-#define DEFAULT_PORT 1234
-#define DEFAULT_IP   "127.0.0.1"
+#define DEFAULT_PORT	    1234
+#define DEFAULT_IP	    "127.0.0.1"
+#define ENCLAVE_CONFIG_TOML "/opt/enclave-tls/config.toml"
```

Comment:
Can you split this PR into several ones? At least I see a global config file for rats-tls is necessary and worthy having a separate PR for it.